src/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React, {
+  forwardRef,
+  ReactNode,
+  FunctionComponent,
+  createContext,
+  HTMLAttributes,
+  DetailedHTMLProps,
+  createElement,
+} from 'react'
+import PropTypes from 'prop-types'
+
+export interface DocumentOutlineSectionContext {
```

Comment:
Does this need to be `export`ed?
`level` and `title` would be read and set by the components in here exclusively, so I think there's no need that the outside world knows about these context properties or the context in general.

Reply:
that's a good point, will drop it